registrations/models.py

```diff
@@ -389,6 +392,7 @@ def can_be_edited_by(self, user):
             user.is_superuser
             or user.is_admin_of(self.publisher)
             or user.is_registration_admin_of(self.publisher)
+            or user.is_substitute_user_of(self.publisher)
```

Comment:
Should this be `or user.is_substitute_user_of(self.registration_user_accesses)`?

Reply:
Indeed it should; this is now fixed.